docker/conda/environments/cuda11.8_dev.yml

```diff
@@ -61,35 +61,41 @@ dependencies:
     - librdkafka=1.7.0
     - mlflow>1.29,<2
     - mrc=23.07
-    - myst-parser==0.17
-    - networkx=2.8
+    - myst-parser==1.0.0
+    - networkx=3.1
     - ninja=1.10
-    - nodejs=17.4.0
+    - nodejs=18.15.0
     - numba>=0.56.2
     - numpydoc=1.4
     - pandas=1.3
     - pip
     - pkg-config # for mrc cmake
     - pluggy=1.0
     - protobuf=4.21.*
+    - pyarrow * *_cuda # Ensure we get a CUDA build. Version determined by cuDF
     - pybind11-stubgen=0.10.5
     - pydot
     - pytest
     - pytest-benchmark>=4.0
     - pytest-cov
     - python-confluent-kafka=1.7.0
     - python-graphviz
-    - python=3.8
+    - python=3.10
     - rapidjson=1.1.0
-    - scikit-build=0.13
-    - scikit-learn=0.23.1
+    - scikit-build=0.17.1
+    - scikit-learn=1.2.2
     - sphinx
     - sphinx_rtd_theme
     - sysroot_linux-64=2.17
     - tqdm=4
     - typing_utils=0.1
     - watchdog=2.1
     - yapf=0.32.0
+
+    # If cuml is installed, we need to constrain these two libraries. See:
+    # https://docs.rapids.ai/notices/rsn0026/ for libcusolver & libcusparse
+    - libcusolver<=11.4.1.48
+    - libcusparse<12
```

Comment:
Is this still true? RSN0026 refers to cudf 22.10. It says nothing of future versions.

Reply:
This was a merge artifact. Looks like that was removed in `branch-23.07` but the merge added it back in for some reason.

Reply:
Fixed in latest commit